### PR TITLE
added basic readme by formatting and appending to @Moshin's Discord thread

### PR DESCRIPTION
**Pull request by mtuckerb**
this isn't complete but it's better than nothing.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
```
Comment (josancamon19) [inline flake.nix]: are this files needed?

Comment (josancamon19) [reply flake.nix]: can we remove them?

**Review by josancamon19 — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 1 above)
Comment (mtuckerb) [reply flake.nix]: we absolutely can, but for anyone using nixos, nix-darwin, or just home-manager, it's incredibly useful to have. For anyone who isn't using those, it's harmless. Your call.

**Comment by mdmohsin7** [comment]
@mtuckerb 
Please replace `fastapi run` with `uvicorn main:app --reload --env-file .env`. This way the users won't have to manually pass any ENV keys

**Comment by mtuckerb** [comment]
Okay, flakes are removed (sorry for all the commit noise) and README is updated with uvicorn.

File: flake.nix (hunk 1 above)
Comment (francip) [reply flake.nix]: Why not add flake.nix to .gitignore, so you don't have to think about removing it in future?

Comment (mtuckerb) [reply flake.nix]: yup. I know how to use gitignore. I thought it might be useful for others who, like me, use nix*. It's been removed though, so can we merge this now?

**Review by josancamon19 — APPROVED** [review]
